packages/models-library/src/models_library/rabbitmq_messages.py

```diff
@@ -19,6 +24,17 @@ class LoggerRabbitMessage(RabbitMessageBase):
     messages: List[str]
 
 
+class EventRabbitMessage(RabbitMessageBase):
+    action: RabbitEventMessageType
+    payload: Optional[Dict[str, Any]] = Field(
```

Comment:
minor: I also do agree that if not used why add it...

Reply:
will remove